sonar-plugin-api/src/main/java/org/sonar/api/measures/MeasuresFilters.java

```diff
@@ -204,6 +204,10 @@ public String filterOnMetricKey() {
     }
   }
 
+  /**
+   * @deprecated since 5.2. The measures related to rules are computed on server side by Compute Engine.
+   */
+  @Deprecated
   private abstract static class AbstractRuleMeasureFilter<M> extends MetricFilter<M> {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)